OpenRA.Mods.Common/Traits/InstantlyRepairable.cs

```diff
@@ -13,20 +13,20 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	public class EngineerRepairType { }
+	public class InstantlyRepairType { }
 
 	[Desc("Eligible for instant repair.")]
-	public class EngineerRepairableInfo : ConditionalTraitInfo
+	public class InstantlyRepairableInfo : ConditionalTraitInfo
 	{
 		[Desc("Actors with these Types under EngineerRepair trait can repair me.")]
-		public readonly BitSet<EngineerRepairType> Types = default;
+		public readonly BitSet<InstantlyRepairType> Types = default(BitSet<InstantlyRepairType>);
```

Comment:
Seems like this fails with a lint error now?
```
Error: /home/runner/work/OpenRA/OpenRA/OpenRA.Mods.Common/Traits/InstantlyRepairable.cs(22,55): error IDE0034: 'default' expression can be simplified [/home/runner/work/OpenRA/OpenRA/OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]
Error: /home/runner/work/OpenRA/OpenRA/OpenRA.Mods.Common/Traits/InstantlyRepairs.cs(24,55): error IDE0034: 'default' expression can be simplified [/home/runner/work/OpenRA/OpenRA/OpenRA.Mods.Common/OpenRA.Mods.Common.csproj]
```
I guess this means this should be
```suggestion
		public readonly BitSet<InstantlyRepairType> Types = default;
```

Reply:
Rebase problems.